### ISSUE TITLE
[Feature]: LSP language configure

### ISSUE THREAD

**Issue by ztby74** [LunarVim/LunarVim#1870]
### Feature motivation

Can you give me a case of configuring a language server in the ?

I finished installing jdtls,   But no response,

Out of the box   Or do I have to configure it according to the official documents,

**Comment by abzcoding** [comment]
for a better java experience you would need to first add this to your `~/.config/lvim/config.lua`

```lua
lvim.plugins = {
    { "mfussenegger/nvim-jdtls", ft = "java" },
}
```

then create the `~/.config/lvim/ftplugin` folder and put `java.lua` file inside it and fill it with this
```lua
-- ~/.config/lvim/ftplugin/java.lua
local status_ok, jdtls = pcall(require, "jdtls")
if not status_ok then
  return
end

local workspace_path = os.getenv "HOME" .. "/workspace/"
local JAVA_LS_EXECUTABLE = os.getenv "HOME" .. "/.local/share/lunarvim/lvim/utils/bin/jdtls"

jdtls.start_or_attach {
  on_attach = require("lvim.lsp").common_on_attach,
  cmd = { JAVA_LS_EXECUTABLE, workspace_path .. vim.fn.fnamemodify(vim.fn.getcwd(), ":p:h:t") },
}

vim.api.nvim_set_keymap("n", "<leader>la", ":lua require('jdtls').code_action()<CR>", { noremap = true, silent = true })

vim.cmd "command! -buffer JdtCompile lua require('jdtls').compile()"
vim.cmd "command! -buffer JdtUpdateConfig lua require('jdtls').update_project_config()"
-- vim.cmd "command! -buffer JdtJol lua require('jdtls').jol()"
vim.cmd "command! -buffer JdtBytecode lua require('jdtls').javap()"
-- vim.cmd "command! -buffer JdtJshell lua require('jdtls').jshell()"
```

and in the end create `~/workspace` folder and it should work well enough

**Comment by abzcoding** [comment]
if everything works out you would get an output like this if you run `:LvimInfo` in a java file
<img width="1752" alt="Screen Shot 2021-10-31 at 7 34 45 PM" src="https://user-images.githubusercontent.com/10992695/139592266-ab871343-efb2-4d3b-8c65-def19b73d09e.png">

**Comment by kylo252** [comment]
Fixed in d046b43acb1247bebf3ef38367ff3372f5169d5e
